modules/ocf/manifests/firewall/post.pp

```diff
@@ -68,4 +70,20 @@
       before      => undef,
     }
   }
+
+
+  # Drop packets on the primary network inteface that are not whitelisted
+  # TODO: eliminate this if statement once testing is complete
+  if $ocf::firewall::drop_other_input {
```

Comment:
This is supposed to be non-optional in the end, right? I don't think it matters at this point, but I would have called it `allow_other_traffic` or something to indicate that the case is exceptional..

Reply:
Sure, let's change it to that. We can rewrite the commit history if it gets too ugly anyway.

Reply:
If it's only test code that's ultimately going away, then I say it's your call whether to rename it.